gtsfm/loader/hilti_loader.py

```diff
@@ -41,11 +42,9 @@
 }
 
 LIDAR_POSE_RELATIVE_PATH = "lidar/fastlio2.g2o"
```

Comment:
Maybe rename this to lidar fastlio relative path? Also, we should be getting absolute poses from the Bayesian ICP too, right?